src/services/provider/topics.provider.js

```diff
@@ -1,5 +1,38 @@
-
+import { $injector } from '../../injection';
+import { Topic } from '../domain/topic';
+/**
+ * @returns {Array} with topics loaded from backend  
+ */ 
 export const loadBvvTopics = async () => {
 	
-	throw new Error('Not yet implemented');
+	const { HttpService: httpService, ConfigService: configService } = $injector.inject('HttpService', 'ConfigService');
+
+	const url = configService.getValueAsPath('BACKEND_URL') + 'topics';
+
+
+	const result = await httpService.fetch(`${url}`, { });
+
+	if (result.ok) {
+		const topics = []; 
+		const payload = await result.json();
+		payload.topics.forEach(definition => {
+			let topic = null;
+			topic = new Topic(
+				definition.id, 
+				definition.label, 
+				definition.description, 
+				definition.defaultBackground, 
+				definition.backgroundLayers, 
+				definition.activatedLayers, 
+				definition.selectedLayers);
+			if (topic.id && topic.label) { //at least the id and the label should be set
```

Comment:
`description` and `backgroundLayers` must also be defined

Reply:
Added check of property description with commit https://github.com/ldbv-by/bav4-nomigration/pull/163/commits/94e069a0c21a34660b3c5961ebc5d399e90ef7fb